R/common-null-pointer.R

```diff
@@ -0,0 +1,40 @@
+#' @title 
+#' Function to find proportion of NAs in each column of a dataframe or matrix
+#' 
+#' @description Finds the proportion of NAs in each column of a dataframe or 
+#' matrix.  NA possibilities that are considered: NA, NULL, NaN, "?", "nil", 
+#' "NA", "NULL", " ", "", "blank","NAs", "na", "??", "empty", "999", "NaN", 
+#' "none", "NONE", "None", "nas". 
+#' @param x A data frame or matrix
```

Comment:
Can you add a second param that gives users the option to add to the list of possible NAs?